### PR TITLE
Hyo tester's reflection on the class so far

### PR DESCRIPTION
**Pull request by hyo9292**
This post is my first embedding an interactive program and using some Markdown for styling.

### CODE REVIEW

**Comment by hyo9292** [comment]
I want to give this another look before it's ready for review: Comment in the meantime are welcome.

File: _post/2022-09-22-hyotester-reflection.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+author: tommytester
+title: "Tommy's Reflection on the Class So Far"
+---
+
+Here's the program I'm embedding:
+
+    <iframe src="https://trinket.io/embed/python/c26c35e489" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+    
+    
+    ```
```
Comment (simon871028) [inline _post/2022-09-22-hyotester-reflection.md:12]: The code block should work properly without the spaces before  ```.

File: _post/2022-09-22-hyotester-reflection.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+author: tommytester
+title: "Tommy's Reflection on the Class So Far"
```
Comment (eah13) [inline _post/2022-09-22-hyotester-reflection.md:4]: Take a look at the author and title.  I believe they should be
~~~
author: hyotester
title: "Hyotester's Reflection on the class so far"
~~~
(or whichever title you choose using your name)

File: _post/2022-09-22-hyotester-reflection.md (hunk 1 above)
Comment (eah13) [reply _post/2022-09-22-hyotester-reflection.md:12]: Good observation! It's better style to put the backticks on the beginning of the line, but I'm not sure whether they'll display on our website like this or not. On Github, they do:
~~~
   ```
Is this formatted like code?
```
~~~
produces:
   ```
Is this formatted like code?
```

How something gets rendered depends on the markdown parser used, a little bit like whether a specific browser can render something.

Comment (eah13) [reply _post/2022-09-22-hyotester-reflection.md:12]: So, in this case I think it'd be a fun experiment to merge this code and see what it looks like.

**Review by eah13 — CHANGES_REQUESTED** [review]
In addition to the author/title update, check the filename: `_post/2022-09-22-hyotester-reflection.md` - our website wouldn't find this, since it's looking in the `_posts` directory instead.

I believe it should be something like `_posts/hyotester/2022-09-22-hyotester-reflection.md` instead.

**Review by greencouchpotato — APPROVED** [review]

**Review by eah13 — APPROVED** [review]

**Pull request merged**